arch/risc-v/src/Makefile

```diff
@@ -24,10 +24,19 @@ ifeq ($(CONFIG_OPENSBI),y)
 include opensbi/Make.defs
 endif
 
+# Kernel runs in supervisor mode or machine mode ?
+
+ifeq ($(CONFIG_ARCH_USE_S_MODE),y)
+ARCH_CMN_MODE_DIR = supervisor
```

Comment:
change to ARCH_MODDIR like ARCH_SRCDIR

Reply:
This macro is not needed so it is removed entirely